pkg/finder/finder_test.go

```diff
@@ -250,3 +250,45 @@ func Test_FileFinderBadPath(t *testing.T) {
 		t.Errorf("Error should be thrown for bad path")
 	}
 }
+
+func Test_fsFinderBadConfig(t *testing.T) {
+	// Missing config file
+	fsFinder := FileSystemFinderInit(
+		StoreConfig("../../test/fixtures/notHere.yaml"),
+	)
+
+	_, err := fsFinder.Find()
+
+	if err == nil {
+		t.Errorf("Error should be thrown for missing Config")
+	}
+
+	// Bad format config file
+	fsFinder = FileSystemFinderInit(
+		StoreConfig("../../test/fixtures/good.yaml"),
+	)
+
+	_, err = fsFinder.Find()
+
+	if err == nil {
+		t.Errorf("Error should be thrown for bad Config data")
+	}
+}
+
+func Test_fsFinderGoodConfig(t *testing.T) {
+	// Proper config file
+	fsFinder := FileSystemFinderInit(
+		WithPathRoots("../../test/fixtures/extensionless"),
+		StoreConfig("../../test/fixtures/config.yaml"),
+	)
+
+	files, err := fsFinder.Find()
+
+	if err != nil {
+		t.Errorf("Error thrown for correct config")
+	}
+
+	if len(files) < 1 {
+		t.Errorf("Unable to find files")
```

Comment:
Here and in other places, there is no need for Errorf, Error is enough

Reply:
Same case as above